tests/test_agent.py

```
@@ -1824,6 +1824,42 @@ def int_func(x: int) -> str:
         Agent('test', output_type=output_type)
 
 
+def test_output_type_native_output_must_be_only():
+    """Test that NativeOutput must be the only output type."""
+    with pytest.raises(UserError, match='`NativeOutput` must be the only output type.'):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)